docs/resources/portal_product_version_classic.md

```diff
@@ -0,0 +1,89 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "konnect_portal_product_version_classic Resource - terraform-provider-konnect"
+subcategory: ""
+description: |-
+  PortalProductVersionClassic Resource
+---
+
+# konnect_portal_product_version_classic (Resource)
+
+PortalProductVersionClassic Resource
+
+## Example Usage
+
+```terraform
+resource "konnect_portal_product_version_classic" "my_portalproductversionclassic" {
```

Comment:
While in v3 there aren't any resources with same name, IMO it is cleaner to name all resources using older API version consistently.